modules/diskq/logqueue-disk-non-reliable.h

```diff
@@ -40,6 +50,8 @@ typedef struct _LogQueueDiskNonReliable
   LogQueueDiskMemoryQueue flow_control_window;
   LogQueueDiskMemoryQueue backlog;
   LogQueueDiskMemoryQueue front_cache_output;
+  gint num_input_queues;
+  InputQueue input_queues[0];
```

Comment:
`_get_length()` should show all messages that are currently in the queue.
Shouldn't we include the number of messages in the input queues?